package.json

```diff
@@ -27,13 +27,12 @@
   "devDependencies": {
     "@types/jest": "^22",
     "express": "^4.16.4",
-    "jest": "^22",
-    "publish": "^0.6.0",
+    "jest": "^24.8.0",
     "ramda": "^0.25.0",
     "react": "16.2.0",
     "react-dom": "16.2.0",
     "react-functional-lifecycle": "^3.0.0",
-    "react-native": "0.50.0",
+    "react-native": "^0.60.0",
```

Comment:
Upgrading RN was not causing any issues for me, might try installing it in a RN project. Also, is the new focus of this repo just for progressive web apps? Should probably get rid of the caret

Reply:
I think ultimately, I'd love for it to work on React Native and web if we're going to have other people using it outside of Aloompa, but we're only using it on the web right now.

Reply:
@tysoncadenhead that would take some testing. Have we used this in a RN app before?